inst/extdata/manifest.csv

```diff
@@ -6,6 +6,7 @@ Reference,Taxonomy,Part,Number,Call
 @baron2016singlecell,10090,pancreas,1886,BaronPancreasData('mouse')
 @bhaduri2020cell,9606,cortical organoids,242349,BhaduriOrganoidData()
 @buettner2015computational,10090,embryonic stem cells,288,BuettnerESCData()
+@bunis2021haematopoietic,9606,haematopoietic stem and progenitor,???,BunisHSPCData
```

Comment:
Where does the manifest's `Number` come from?  Is it a cell number or taxonomy item?  And if cell number, the unfiltered number?

Reply:
Just give the number of cells that you get from running `BunisHSPCData()` with the default arguments. Note that the last thing is the actual command (it'll get wrapped in backticks when it gets printed in the vignette).